flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/context/SessionContext.java

```diff
@@ -275,4 +315,33 @@ private void resetSessionConfigurationToDefault(Configuration defaultConf) {
         }
         sessionConfiguration.addAll(defaultConf);
     }
+
+    private void buildClassLoaderAndUpdateDependencies(Collection<URL> newDependencies) {
+        // merge the jar in config with the jar maintained in session
+        Set<URL> jarsInConfig;
+        try {
+            jarsInConfig =
+                    new HashSet<>(
+                            ConfigUtils.decodeListFromConfig(
+                                    sessionConfiguration, PipelineOptions.JARS, URL::new));
+        } catch (MalformedURLException e) {
+            throw new SqlExecutionException(
+                    "Failed to parse the option `JARS` in configuration.", e);
```

Comment:
Please use the value of `PipelineOptions.JARS` instead of `JARS`.

Reply:
Regarding "value of `PipelineOptions.JARS`", I mean `"pipeline.jars"`. I think no user can understand what `PipelineOptions.JARS` refers, that is an internal field.